src/main/resources/de/tracetronic/jenkins/plugins/ecutest/tool/StartTSBuilder/help.html

```diff
@@ -36,6 +39,10 @@
 def instance = ET.newInstallation('ECU-TEST', 'C:\\Program Files\\ECU-TEST 8.0')
 instance.startTS()
             </pre>
+            <pre>
+def instance = ET.newInstallation toolName: 'ECU-TEST', installPath: C:\\Program Files\\ECU-TEST 8.0'
+instance.startTS toolLibsIniPath: C:\\ToolLibs.ini, tcpPort: 5017, timeout: '60', keepInstance: false
```

Comment:
timeout is small integer, would it work with quotas? `'60'`

Reply:
Timeout property can be both, String or int.
\@DataBoundSetters exist for both types.